core/server/models/post.js

```diff
@@ -157,7 +166,7 @@ Post = ghostBookshelf.Model.extend({
                 return Promise.reject(new errors.ValidationError(
                     i18n.t('errors.models.post.expectedPublishedAtInFuture')
                 ));
-            } else if (moment(publishedAt).isBefore(moment().add(5, 'minutes'))) {
+            } else if (this.hasChanged('published_at') && moment(publishedAt).isBefore(moment().add(2, 'minutes'))) {
```

Comment:
just putting it into the config will not bring any value in my opinion?(maybe for testing purposes, but i didn't need it)
i could think of returning it the ember, so both will use the same number?but no prio right now.